lib/trino-parquet/src/main/java/io/trino/parquet/reader/ColumnReaderFactory.java

```diff
@@ -111,6 +111,7 @@ private static List<ColumnReaderProvider> getColumnReaderProviders(PrimitiveType
                     createDecimalColumnReaderProvider(),
                     createProvider(BinaryColumnReader::new));
             case FIXED_LEN_BYTE_ARRAY -> ImmutableList.of(
+                    LongFlatColumnReader.shortDecimalReaderProvider(),
```

Comment:
nit: static import

Reply:
It feels useful to keep the "Long" prefix as it makes it easier to see what type of Block the output will be